test/extended/operators/operators.go

```diff
@@ -101,68 +101,65 @@ var _ = g.Describe("[sig-arch][Early] Managed cluster should", func() {
 				return false, nil
 			}
 
-			var unavailable []objx.Map
-			var unavailableNames []string
+			var working []string
 			for _, co := range items {
-				if condition(co, "Available").Get("status").String() != "True" {
-					ns := co.Get("metadata.namespace").String()
-					name := co.Get("metadata.name").String()
-					unavailableNames = append(unavailableNames, fmt.Sprintf("%s/%s", ns, name))
-					unavailable = append(unavailable, co)
-					break
-				}
-				if condition(co, "Progressing").Get("status").String() != "False" {
-					ns := co.Get("metadata.namespace").String()
-					name := co.Get("metadata.name").String()
-					unavailableNames = append(unavailableNames, fmt.Sprintf("%s/%s", ns, name))
-					unavailable = append(unavailable, co)
-					break
-				}
-				if condition(co, "Failing").Get("status").String() != "False" {
-					ns := co.Get("metadata.namespace").String()
-					name := co.Get("metadata.name").String()
-					unavailableNames = append(unavailableNames, fmt.Sprintf("%s/%s", ns, name))
-					unavailable = append(unavailable, co)
-					break
+				conditions, missingTypes := surprisingConditions(co)
+				if len(conditions) > 0 || len(missingTypes) > 0 {
+					working = append(working, co.Get("metadata.name").String())
```

Comment:
```suggestion
					unready = append(unready, co.Get("metadata.name").String())
```
working is a confusing name here.  i first read it as "these are the operators that are functional" not "these are the operators that are still churning"

Reply:
Fixed in fe2a717 -> 5819963.